### ISSUE TITLE
Class Support for Outline (Nested Functions within Class declarations)

### ISSUE THREAD

**Issue by BloodyRnar** [mark-wiemer/vscode-autohotkey-plus-plus#29]
Outline view shows all functions, even if they are within a Class, but does not group/label by Class.

![image](https://user-images.githubusercontent.com/19176329/104989799-d4a40f00-59e0-11eb-96a4-f75b84cd12a3.png)

Would be amazing to have the outline handle this!

`; Test Class Description
Class TestClass {
    ; __New() - Constructor
    __New() {
        ; Constructor
    }

    ; Simple Test Function
    AFunctionToTest() {
       ; Maybe do something useful.... 
    }

}
`

**Comment by mark-wiemer** [comment]
Yep, this would be a great feature. The priority right now is bugfixing, documentation, and adding automated tests. I hope you'll understand that I likely won't add this for a bit :/ , but thanks for your suggestion
